src/replicates.jl

```diff
@@ -1,36 +1,25 @@
 """ This file includes functions to calculate values related to replicates. """
 
 ### A function to predict G1 and G2 for the three replicates.###
-function predict_replicates(p1, p2, p3, concs1, g0)
-
+function predict_replicates(p1, p2, p3, g0)
     t = LinRange(0.0, 95.0, 189)
-    G1_1 = ones(189, 8)
-    G2_1 = ones(189, 8)
-    G1_2 = ones(189, 8)
-    G2_2 = ones(189, 8)
-    G1_3 = ones(189, 8)
-    G2_3 = ones(189, 8)
+    ps = cat(p1, p2, p3, dims = 3)
+    G1 = ones(189, 8, 3)
+    G2 = ones(189, 8, 3)
 
-    for i = 1:8 # concentration number
-        G1_1[:, i], G2_1[:, i], _ = predict(p1[:, i], g0, t)
-        G1_2[:, i], G2_2[:, i], _ = predict(p2[:, i], g0, t)
-        G1_3[:, i], G2_3[:, i], _ = predict(p3[:, i], g0, t)
+    for i = 1:size(ps, 2) # concentration number
+        for j = 1:size(ps, 3)
+            G1[:, i, j], G2[:, i, j], _ = predict(p1[:, i, j], g0, t)
+        end
     end
 
```

Comment:
Oh sorry, there is one. `Error: BoundsError: attempt to access 9×@*(8 Array(*@{Float64,2} at index [1:9, 1
, 2]` there is an error in the `replicatesAtOnce.jmd` which I think may refer to this part. Maybe the `effs1,2,3` that is passed to the `predict_replicate` is not 3dimensions?